sfcasts/ep1/hungry-dinos.md

```diff
@@ -1,75 +1,78 @@
-accepting visitors and our health status will be hungry. And we want to assert that
-hungry donors are accepting visitors. All right, let's come back up here to our test
-method. And we will tell this test method to use the data provider. So at data
-provider, and we're going to use the health status provider.
-
-All right, we can go ahead and rename this method too, because test is not accepting
-visitors of sick. It's not really doing that anymore. Let's change that to test is
-accepting visitors based on health status. And we will pass in a health status
-argument, and this is going to be health status. And then we're also going to pass in
-a ion and this will be our expected visitor status. Awesome. All right, let's fix up
-our test here so we can continue use bumpy for a dinosaur and this now, instead of
-sitting the set, the health, uh, sick, oops, we want to set the health to our health
-status that our provider is giving us. And then assert false is accepting visitors.
-Let's change this to selfer saying that expected health status is the same as Dino is
-accepting visitors. All right, move into your terminal type in vendor bin PHP unit.
-But this time we're going to show you something a little bit different. Cool, little
-trick PHP unit can do pass in a filter flag. And for the argument type in test is
-accepting visitors based on health status. Enter, check this out. We have two tests
-and two assertions in one failure. Our dinosaur test test is accepting visitors based
-on health status with our hungry <inaudible> is not accepting visitors. Uh, data set
-failed, asserting that false is identical to true.
-
-We'll fix this in just a second, but this filter flag that we provided up here, uh,
-to PHP unit, what this is doing, it's telling PHP unit that we only want to run tests
-that have tests is accepting visitors based on health status. As part of their name,
-let's go back and into dinosaur class so we can fix our accepting visitors method.
-And we'll scroll down to is accepting visitors and ha there's the problem. This
-health is identical to health status healthy. Mm let's. Change this to this health
-status is not identical to health status sick. Then we return true. Otherwise we
-return false. Alright, let's move back into our browser and vendor been PhD unit.
-We're going to do our filter plug again, but aside from, uh, passing in test method,
-names or test classes, we can also do hungry. I know is accepting visitors. Great. We
-have one test and one assertion. This was the test that was failing. As you can see,
-we fixed two problems. We can, we fixed the two problems, one our browsers up and
-running. Again, two, we have passing tests and three we've learned how to use the
-filter flag with PHP unit, just to make sure that our tests are back up and running.
-Let's go ahead and run vendor and PHP unit
-
-And great all nine tests. And the 14 assertions are passing..
+# Filtering Out Hungry Dino's
```

Comment:
We need to modify the "closing" of the previous chapter. We say:

>In our HealthStatus enum, we don't have a case for Hungry status labels. Go figure. Is a hungry dinosaur accepting visitors? I don't know - I guess it depends on if you ask the visitor or the dino. Anyways, Hungry is not a status we expected. So next, let's throw a clear exception if we run into an unknown status and test for that exception.

But we dont actually do that until the next chapter